Tests/RequestTests.swift

```diff
@@ -0,0 +1,211 @@
+//
+//  RequestTests.swift
+//  HyperspaceTests
+//
+//  Created by Tyler Milner on 6/26/17.
+//  Copyright © 2017 Bottle Rocket Studios. All rights reserved.
+//
+
+import XCTest
+import Hyperspace
+
+class RequestTests: XCTestCase {
+    
+    // MARK: - Tests
+    
+    func test_SimpleGETRequestWithoutQueryParametersOrHeaders_GeneratesCorrectURLRequest() {
+        let request: Request<String, AnyError> = .simpleGET
+        assertParameters(for: request)
+    }
+    
+    func test_SimpleGETRequestWithHeaders_GeneratesCorrectURLRequest() {
+        var request: Request<String, AnyError> = .simpleGET
+        request.headers = [.contentType: .applicationJSON]
+        
+        assertParameters(headers: ["Content-Type": "application/json"], for: request)
+    }
+    
+    func test_SimplePOSTRequestWithData_GeneratesCorrectURLRequest() {
+        let bodyData = "Test".data(using: .utf8)!
+
+        var request: Request<String, AnyError> = .simplePOST
+        request.body = HTTP.Body(bodyData)
+        
+        assertParameters(method: "POST", body: bodyData, for: request)
+    }
+    
+    func test_Request_EmptyResponseInit() {
```

Comment:
I know you didn't write this, but let's go ahead and remove this test since it's not actually useful.

Reply:
Fixed